scripts/data.js

```diff
@@ -1,42 +1,42 @@
-var myProjects = [
+[
   {
-    title: 'Campfire Coffee',
-    completedOn: '2016-10-10',
-    numFiles: 4,
-    ghPages: 'https://dkulp23.github.io/campfire_coffee/',
-    imageURL: '/assets/campfire-coffee.png',
-    imgAlt: '',
-    projectType: 'business',
-    description: '<p>Campfire Coffee was our introduction to the use of the JavaScript constructor function...</p>',
+    "title": "Campfire Coffee",
+    "completedOn": "2016-10-10",
+    "numFiles": 4,
+    "ghPages": "https://dkulp23.github.io/campfire_coffee/",
+    "imageURL": "/assets/campfire-coffee.png",
+    "imgAlt": "",
+    "projectType": "business",
+    "description": "<p>Campfire Coffee was our introduction to the use of the JavaScript constructor function...</p>",
   },
   {
-    title: 'Guessing Game',
-    completedOn: '2016-10-05',
-    numFiles: 3,
-    ghPages: 'https://dkulp23.github.io/guessing_game/',
-    imageURL: '/assets/guessing-game.png',
-    imgAlt: '',
-    projectType: 'game',
-    description: '<p>Guessing Game was our very first 201 project...</p>',
+    "title": "Guessing Game",
+    "completedOn": "2016-10-05",
+    "numFiles": 3,
+    "ghPages": "https://dkulp23.github.io/guessing_game/",
+    "imageURL": "/assets/guessing-game.png",
+    "imgAlt": "",
+    "projectType": "game",
+    "description": "<p>Guessing Game was our very first 201 project...</p>",
   },
   {
-    title: 'Bus Mall',
-    completedOn: '2016-10-20',
-    numFiles: 5,
-    ghPages: 'https://dkulp23.github.io/bus-mall/',
-    imageURL: '/assets/bus-mall.png',
-    imgAlt: '',
-    projectType: 'game',
-    description: '<p>Bus Mall was a fun project that allowed us to practice object oriented programming...</p>',
+    "title": "Bus Mall",
+    "completedOn": "2016-10-20",
+    "numFiles": 5,
+    "ghPages": "https://dkulp23.github.io/bus-mall/",
+    "imageURL": "/assets/bus-mall.png",
+    "imgAlt": "",
+    "projectType": "game",
+    "description": "<p>Bus Mall was a fun project that allowed us to practice object oriented programming...</p>",
   },
   {
-    title: 'SOS',
-    completedOn: '2016-11-01',
-    numFiles: 10,
-    ghPages: 'https://dkulp23.github.io/sos/',
-    imageURL: '/assets/sos.png',
-    imgAlt: '',
-    projectType: 'services',
-    description: '<p>SOS is a project that my team and I were very proud of...</p>',
+    "title": "SOS",
+    "completedOn": "2016-11-01",
+    "numFiles": 10,
+    "ghPages": "https://dkulp23.github.io/sos/",
+    "imageURL": "/assets/sos.png",
+    "imgAlt": "",
+    "projectType": "services",
+    "description": "<p>SOS is a project that my team and I were very proud of...</p>",
   }
-];
+]
```

Comment:
You should delete your data.js file completely because now you're just using your json file :)